ton_client/client/src/build_info.rs

```diff
@@ -0,0 +1,18 @@
+/*
+* Copyright 2018-2020 TON DEV SOLUTIONS LTD.
+*
+* Licensed under the SOFTWARE EVALUATION License (the "License"); you may not use
+* this file except in compliance with the License.
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific TON DEV software governing permissions and
+* limitations under the License.
+*/
+use crate::dispatch::DispatchTable;
+
+pub(crate) fn register(handlers: &mut DispatchTable) {
```

Comment:
We can't read files in runtime.
The json content must be embedded into rs source as a string constant.